tests/core/v5_1/alexandria/test_partial_proofs.py

```diff
@@ -37,10 +37,10 @@ def test_ssz_full_proofs(data):
     assert len(proven_data_segments) == 1
     start_index, proven_data = proven_data_segments[0]
     assert start_index == 0
-    assert proven_data == data
+    assert proven_data == content
```

Comment:
👍 for consistency, but are you making a distinction between `content` and `data`? e.g., why `get_proven_data` isn't `get_proven_content`?

Reply:
My mental model has been: `content` is comprised of `data` so trying to only use `content` to mean the full thing.